builder/Dockerfile

```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM golang:1.19
+FROM golang:1.19.7
```

Comment:
kindly also update the golang version in the dockerfile we use to build autoscaler image
https://github.com/elankath/autoscaler/blob/sync-rel-v1.26/cluster-autoscaler/Dockerfile

Reply:
fixed